src/components/header/ConnectionIndicator.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="wrapper--indicator">
+    <div class="dot" :class="getDotClass(connectionType)"></div>
```

Comment:
![Screen Shot 2022-03-31 at 4 24 38 PM](https://user-images.githubusercontent.com/40356749/161078700-c9535559-3b05-4928-abf2-3b11a9122ffb.png)

The indicator position is rendered in a strange way

Reply:
![스크린샷 2022-04-01 오전 11 50 08](https://user-images.githubusercontent.com/1606820/161185774-7072c7cc-6055-4175-b530-edd9dd8c0189.png)

that's quite weird. `wrapper--indicator` style is ignored in staging